lineapy/plugins/argo_pipeline_writer.py

```diff
@@ -0,0 +1,244 @@
+import base64
+import errno
+import logging
+import os
+from enum import Enum
+from typing import Any, Dict, List, Optional
+
+from kubernetes import client, config
+from typing_extensions import TypedDict
+
+from lineapy.plugins.base_pipeline_writer import BasePipelineWriter
+from lineapy.plugins.task import (
+    DagTaskBreakdown,
+    TaskDefinition,
+    TaskGraph,
+    TaskSerializer,
+    render_task_io_serialize_blocks,
+)
+from lineapy.plugins.taskgen import get_task_definitions
+from lineapy.plugins.utils import load_plugin_template
+from lineapy.utils.logging_config import configure_logging
+from lineapy.utils.utils import prettify
+
+logger = logging.getLogger(__name__)
+configure_logging()
+
+
+class ARGODagFlavor(Enum):
+    PythonOperatorPerSession = 1
```

Comment:
Rename and uncomment the ununsed ones to match Argo's language. 

I believe Argo calls a component a "Step", so StepPerSession and StepPerArtifact